src/superphot_plus/ztf_transient_fit.py

```diff
@@ -143,26 +165,25 @@ def create_prior(cube):
         np.ndarray
             Updated array of parameters.
         """
+        # Compute the truncated Gaussian distribution for all values at once.
+        tg_vals = truncnorm.ppf(cube, tg_a, tg_b, loc=prior_mean, scale=prior_std)
 
-        cube[0] = max_flux * 10 ** (
-            trunc_gauss(cube[0], *PRIOR_A)
-        )  # log-uniform for A from 1.0x to 16x of max flux
-        cube[1] = trunc_gauss(cube[1], *PRIOR_BETA)  # beta UPDATED, looks more Lorentzian so widened by 1.5x
-        cube[2] = 10 ** trunc_gauss(cube[2], *PRIOR_GAMMA)  # very broad Gaussian temporary solution for gamma
-        max_flux_loc = tdata[np.argmax(fdata[bdata == "r"] - np.abs(ferrdata[bdata == "r"]))]
-        cube[3] = trunc_gauss(cube[3], np.amin(tdata) - 50.0, np.amax(tdata) + 50.0, max_flux_loc, 20.0)  # t0
-        cube[4] = 10 ** (trunc_gauss(cube[4], *PRIOR_TAU_RISE))  # taurise, UPDATED
-        cube[5] = 10 ** (trunc_gauss(cube[5], *PRIOR_TAU_FALL))  # tau fall UPDATED
-        cube[6] = 10 ** (trunc_gauss(cube[6], *PRIOR_EXTRA_SIGMA))  # lognormal for extrasigma, UPDATED
+        cube[0] = max_flux * 10 ** (tg_vals[0])  # log-uniform for A from 1.0x to 16x of max flux
+        cube[1] = tg_vals[1]  # beta UPDATED, looks more Lorentzian so widened by 1.5x
+        cube[2] = 10 ** (tg_vals[2])  # very broad Gaussian temporary solution for gamma
+        cube[3] = tg_vals[3]
+        cube[4] = 10 ** (tg_vals[4])  # taurise, UPDATED
+        cube[5] = 10 ** (tg_vals[5])  # tau fall UPDATED
+        cube[6] = 10 ** (tg_vals[6])  # lognormal for extrasigma, UPDATED
 
         # green band
```

Comment:
It's not *really* green.

```suggestion
        # g band
```